test/unittests/parser_test.cpp

```diff
@@ -11,10 +11,30 @@ static const auto functype_i32_to_void = from_hex("60017f00");
 
 namespace
 {
-bytes make_section(uint8_t id, const bytes& content)
+bytes prefix_with_size(const bytes& content)
 {
     assert(content.size() < 0x80);
-    return bytes{id, static_cast<uint8_t>(content.size())} + content;
+    return bytes{static_cast<uint8_t>(content.size())} + content;
```

Comment:
@chfast wanted to specifically go against using `uint8_t`  with the plus operator in these cases.

Reply:
I mostly wanted avoid having `uint8_t + uint8_t` producing bytes. Here is fine, because not this case and inside a helper. But the previous version is fine as well.

Reply:
I'll just keep it as is for now.